pareto/examples/strategic_model_treatment_module_demo/strategic_model_treatment_module_demo.ipynb

```diff
@@ -0,0 +1,1662 @@
+    "\n",
+    "#### NRA: node-to-treatment arcs\n",
+    "![NRA arcs](data_input_NRA.png)\n",
+    "\n",
+    "#### RNA: treatment-to-node arcs\n",
+    "![RNA arcs](data_input_RNA.png)\n",
+    "\n",
+    "NOTE: Support for RNA arcs is likely to be removed in a future version of PARETO.\n",
+    "\n",
+    "#### RSA: treatment-to-storage arcs\n",
+    "![RSA arcs](data_input_RSA.png)\n",
+    "\n",
+    "#### RCA: treatment-to-completions arcs\n",
+    "![RCA arcs](data_input_RCA.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Initial treatment capacity\n",
+    "In tab \"InitialTreatmentCapacity\", specify the initial treatment capacity at each site:\n",
+    "\n",
+    "![Initial treatment capacity](data_input_initial_treatment_capacity.png)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Treatment operational cost (OPEX)\n",
+    "In tab \"TreatmentOperationalCost\", specify the operational cost of each technology at each site:\n",
+    "\n",
+    "![Treatment OPEX](data_input_treatment_OPEX.png)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Treatment expansion cost (CAPEX)\n",
+    "In tab \"TreatmentExpansionCost\", specify the cost of building additional capacity of each technology at each site:\n",
+    "\n",
+    "![Treatment expansion cost](data_input_treatment_expansion_cost.png)\n",
+    "\n",
+    "NOTE: Only one treatment technology can be selected for expansion at each treatment site."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Treatment water efficiency (water recovery)\n",
+    "In tab \"TreatmentEfficiency\", specify the water efficiency of each technology at each site (this parameter amounts to the fraction of water in the inlet which is recovered in the treated stream):\n",
+    "\n",
+    "![Treatment efficiency](data_input_treatment_efficiency.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Pad water quality\n",
+    "In tab \"PadWaterQuality\", specify the concentration of each quality component in the produced water which comes from each production pad and completions pad:\n",
+    "\n",
+    "![Pad water quality](data_input_pad_water_quality.png)\n",
+    "\n",
+    "IMPORTANT: The column header(s) in this tab are used to specify the quality components that are part of the model. That is, there is not a separate tab to specify the quality components. In the above screenshot, there is only one quality component (TDS)."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Component removal effiency\n",
+    "In tab \"RemovalEfficiency\", for each, site, technology, and quality component, specify the fraction of the component (by mass) which goes to the residual stream:\n",
```

Comment:
A recall comment to differentiate removal efficiency from the treatment efficiency defined above may be useful here

Reply:
Addressed.